apps/console/src/features/applications/components/application-danger-zone.tsx

```diff
@@ -96,11 +98,12 @@ export const ApplicationDangerZoneComponent: FunctionComponent<ApplicationDanger
                 setShowDeleteConfirmationModal(false);
                 onDelete();
 
-                eventPublisher.publish("application-delete", {
-                    "client-id": clientId
-                });
+                eventPublisher.publish(
+                    "application-delete", 
+                    { "client-id": clientId }
+                );
             })
-            .catch((error) => {
+            .catch((error: AxiosError) => {
```

Comment:
Can we check if this is `IdentityAppsApiException`?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3829#:~:text=Fixed%20error%20type-,3030a76,-ashanthamara%20dismissed%20dasuni